### ISSUE TITLE
Add note about scroll-behaviour: smooth to readme

### ISSUE THREAD

**Issue by kapoko** [Stanko/animated-scroll-to#55]
Having [scroll-behaviour](https://www.w3schools.com/cssref/pr_scroll-behavior.asp) set to smooth in css:
```scss
:root {
  scroll-behaviour: smooth
}
```
is breaking / delaying this plugins' animation (in Chrome). Found out after some trial and error that this is the default in `bootstrap-5.0.0@beta2`. Same thing is happening with [similar](https://www.npmjs.com/package/scroll-to) [packages](https://www.npmjs.com/package/scroll-to-element).

Example: https://codepen.io/kapoko/pen/rNWbXxN

This setting is probably gonna be used more often. You think it's worth mentioning in the docs?

**Comment by Stanko** [comment]
Hello @kapoko,
That is a good catch! I'll add it to the documentation. I'll also check if it can be detected via javascript to print a warning in the console.

Cheers!

**Comment by Stanko** [comment]
It seems it can be detected like this:
```js
getComputedStyle(document.documentElement).getPropertyValue('scroll-behavior') // -> "smooth"
```
I'll test it more to check cross browser support and release a new version this week.

**Comment by Stanko** [comment]
Fixed in `2.2.0`, I just published it 🎉 

Code:
https://github.com/Stanko/animated-scroll-to/blob/master/src/animated-scroll-to.ts#L196-L203

Tests:
https://github.com/Stanko/animated-scroll-to/blob/master/cypress/integration/scroll-element.js#L116-L132
https://github.com/Stanko/animated-scroll-to/blob/master/cypress/integration/scroll-window.js#L130-L147
